model/accounts_mgmt/v1/subscriptions_resource.model

```diff
@@ -82,6 +82,12 @@ resource Subscriptions {
         // If true, includes the labels on a subscription in the output. Could slow request response time.
         in fetchLabels Boolean
 
+        // If true, includes the capabilities on a subscription in the output. Could slow request response time.
+        in fetchCapabilities Boolean
+
+        // If true, includes the capabilities on a subscription in the output. Could slow request response time.
+        in fetchCpuAndSocket Boolean
+
```

Comment:
The indentation is off, and parameter names should start with upper case characters, so these should be `Fetch...`. Please fix that before merging.

Note also that the actual names expected by the SDK will use the equivalent "snake case": `fetch_labels`, `fetch_capabilities` and `fetch_cpu_and_socket`. Make sure that your server implementation understands that.